src/styles/components/widgets/_button.scss

```diff
@@ -51,7 +51,9 @@ button {
       padding-left: 0;
     }
   }
-
+  &--isMenu {
+    @include button(menu);
```

Comment:
This looks like it might not actually be doing anything. I don't see any edits to the mixin for this, `@mixin button{ ... }` found in the `_mixins.scss` partial.